mlrun/runtimes/function.py

```diff
@@ -175,7 +176,8 @@ def __init__(
         self.function_kind = function_kind
         self.source = source or ""
         self.config = config or {}
-        self.function_handler = ""
+        self.function_handler = None
+        self.function_runtime = None
```

Comment:
Why do you need the word function ? it's a function class, everything under it is of a function

Reply:
`function_handler` was there already, i can change to use the `default_handler` like in jobs vs add a new one (will remove the if/else below), the `runtime` may want to say `nuclio_runtime` not to confuse w mlrun runtime

Reply:
Oh didn't notice `function_handler` was already there, so I guess leave as is
and yeah I guess `nuclio_runtime` makes more sense